src/Compilers/Extension/CompilerExtension.csproj

```diff
@@ -46,7 +46,7 @@
     <NuGetPackageToIncludeInVsix Include="System.Collections.Concurrent" />
     <NuGetPackageToIncludeInVsix Include="System.Collections.Immutable" />
     <NuGetPackageToIncludeInVsix Include="System.Console" />
-    <NugetPackageToIncludeInVsix Include="System.Diagnostics.Contracts" />
+    <NuGetPackageToIncludeInVsix Include="System.Diagnostics.Contracts" />
```

Comment:
Is this a functional or style consistency change? If it's the former then I may update BuildBoss to catch it.

Reply:
Style consistency.